<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>581</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_vehicule</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom_beneficiaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>model</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>num_chassis</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prix_vente</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_vehicule">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_beneficiaire">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_type">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_model">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_chassis">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix_vente">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>afficher </string>
     </attribute>
     <widget class="QTableView" name="tab_vente">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QLabel" name="id_vehicule">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>461</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>93</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_vehicule</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nom_beneficiaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>model</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>num_chassis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_modif">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="beni_modif">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="type_modif">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="model_modif">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_modif">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prix_modif">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
